### PR TITLE
Gutenboarding: Align suggestions under verticals container

### PR DESCRIPTION
**Pull request by chriskmnds**
#### Changes proposed in this Pull Request

Handles #38713  🤞 

Before: 

Screenshot 2020-01-08 at 3 26 15 PM

After:

Screenshot 2020-01-08 at 3 26 33 PM

### CODE REVIEW

**Comment by simison** [comment]
Nice fix!

File: client/landing/gutenboarding/onboarding-block/vertical-select/style.scss [hunk 1]
```diff
@@ -1,6 +1,14 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 
 .vertical-select {
+	width: 400px;
+}
+
+.vertical-select__suggestions {
+	width: inherit;
+	max-width: inherit;
+	margin-left: 0.2em;
```
Comment (chriskmnds) [inline client/landing/gutenboarding/onboarding-block/vertical-select/style.scss]: Supposedly this is the 8px left margin removed from below. `0.2em` is the margin used in question component and updated here to match 🤔  https://github.com/Automattic/wp-calypso/blob/207ba00b4a818ac13cf2d32a23374556ca81aa5d/client/landing/gutenboarding/onboarding-block/question/style.scss#L32.

**Comment by ockham** [comment]
Hmm, this isn't looking so good on mine :confused: 

image

**Comment by chriskmnds** [comment]
> Hmm, this isn't looking so good on mine 😕
> 
> image

Yeah, not surprised. I had a feeling this wouldn't work outside Chrome - just no time to test yesterday.

**Comment by ockham** [comment]
FWIW, this is actually on Chromium -- which for all practical purposes should render identically to Chrome...

**Comment by chriskmnds** [comment]
> FWIW, this is actually on Chromium -- which for all practical purposes should render identically to Chrome...

Not sure what the situation is with Chromium, but it's Chrome that I am using for development. In any case, this was behaving like before in Safari, but fixed with latest commit.

I am on Mac OS 10.14.16 - and this looks identical in Safari 13.0.4, Chrome 79, Firefox 66:

Screenshot 2020-01-10 at 8 48 47 PM


Not planning to merge now - need to update the parent branch and cleanup/update after.

**Comment by chriskmnds** [comment]
Thanks for reviewing @ockham ! I've updated after cleaning up the base branch. There should be no regressions. :-)

**Comment by sirreal** [comment]
Does this need work other than a rebase and fixing up the tests?

I'll add @jeraldjuice @Addison-Stavlo in case they can get this landed.

**Comment by ockham** [comment]
> Does this need work other than a rebase and fixing up the tests?

Flip the order :upside_down_face: : Tests should be fixed in the branch that this PR is based on (#38716; I [think](https://github.com/Automattic/wp-calypso/pull/38717#issuecomment-575155594) it's just an outdated snapshot); once that's done (and merged), this one can be rebased.

> I'll add @jeraldjuice @Addison-Stavlo in case they can get this landed.

That'd be much appreciated, while @chriskmnds and I are messing about with the Gutenberg plugin update.

**Comment by Addison-Stavlo** [comment]
Rebased, handled some merge conflicts, and updated the snapshot.
I think things are good, but could use a sanity check.

**Pull request merged**